scripts/prepare_sector_network.py

```diff
@@ -1826,10 +1826,18 @@ def add_heat(n, costs):
     ]
 
     cop = {
-        "air": xr.open_dataarray(snakemake.input.cop_air_total)
+        "air decentral": xr.open_dataarray(snakemake.input.cop_air_decentral_heating)
         .to_pandas()
         .reindex(index=n.snapshots),
-        "ground": xr.open_dataarray(snakemake.input.cop_soil_total)
+        "ground decentral": xr.open_dataarray(
+            snakemake.input.cop_soil_decentral_heating
+        )
+        .to_pandas()
+        .reindex(index=n.snapshots),
+        "air central": xr.open_dataarray(snakemake.input.cop_air_central_heating)
+        .to_pandas()
+        .reindex(index=n.snapshots),
+        "ground central": xr.open_dataarray(snakemake.input.cop_soil_central_heating)
         .to_pandas()
         .reindex(index=n.snapshots),
     }
```

Comment:
just a thought, but maybe one can build a dictionary with filenames first and then do:

```py
cop = {k: xr.open_dataarray(fn).to_pandas().reindex(index=n.snapshots) for k, vn in cop.items()}
```

which would reduce code length

Reply:
I agree. We could replace `soil` with `ground` everywhere (or vice versa), define something like `heat_pump_sources = ["air", "ground"]`, `heat_system_type=["central", "decentral"]`, maybe in `config.default`, and use dict comprehensions over `heat_pump_sources` & `heat_system_type` wherever possible. What do you think?

Reply:
Yes! I would vote for keeping `ground` and replacing `soil` since it changes less in the config files users are exposed to.